ci/Jenkinsfile.nightly.integration

```
@@ -30,36 +30,44 @@ pipeline {
   }
 
   stages {
-    stage('Build') {
+    stage('Build waku') {
       steps {
         /* Node binary is required for integration tests */
-        sh 'cargo build'
+        sh 'cargo build -p nomos-node --feature waku'
```

```suggestion
        sh 'cargo build -p nomos-node --features waku'
```
i think this is a typo